package.json

```diff
@@ -55,24 +55,24 @@
     "TypeScript"
   ],
   "dependencies": {
-    "@typescript-eslint/parser": "^4.0.0",
+    "@typescript-eslint/parser": "^4.15.0",
     "eslint-config-standard": "^16.0.0"
   },
   "peerDependencies": {
+    "@typescript-eslint/eslint-plugin": ">=4.15.0",
     "eslint": ">=7.12.1",
     "eslint-plugin-import": ">=2.22.1",
     "eslint-plugin-node": ">=11.1.0",
     "eslint-plugin-promise": ">=4.2.1",
-    "typescript": ">=3.9",
-    "@typescript-eslint/eslint-plugin": ">=4.0.1"
+    "typescript": ">=3.9"
   },
   "devDependencies": {
     "@commitlint/cli": "12.1.4",
     "@commitlint/config-conventional": "12.1.4",
     "@commitlint/travis-cli": "12.1.4",
     "@types/eslint": "7.2.13",
     "@types/node": "14.17.3",
-    "@typescript-eslint/eslint-plugin": "4.0.1",
+    "@typescript-eslint/eslint-plugin": "4.15.0",
```

Comment:
I'd like dependency upgrades in a separate PR, even if they are necessary for this one, please.

Reply:
PR available there: #592

Reply:
#592 failed checks due to one of the dependency constraints changing in `standard` that this code base originally take into account. I fixed it here, so hope that won't be an issue.

Reply:
Actually, it will be fixed with #590.